benchmark/benchmark.py

```diff
@@ -101,7 +101,7 @@ def summarize(run_dir, metrics, expand_metrics=False):
         # post-processing of report: show a few selected/important metric
         for metric in metrics:
             keys = metric.split(".")
-            value = report
+            value = report.to_dict()
```

Comment:
v0.2.0 doesn't need `to_dict`, but `0.4.0` needs this.

Reply:
v0.2.0 of what package?

Reply:
sorry, it's `optimum-benchmark`

Reply:
The issue is specifically from this `__post_init__` in v0.2.0 that was incomplete ; only casts the config dict to the appropriate dataclass:
https://github.com/huggingface/optimum-benchmark/blob/v0.2.0/optimum_benchmark/base.py#L25-L29
which was fixed/completed in:
https://github.com/huggingface/optimum-benchmark/blob/v0.3.0/optimum_benchmark/benchmark/base.py#L24-L33